operations/patch.go

```diff
@@ -102,6 +102,13 @@ func Patch() cli.Command {
 				return err
 			}
 
+			if params.Description == "" {
+				params.Description, err = getDefaultDescription()
+				if err != nil {
+					grip.Debug(err)
```

Comment:
shouldn't this be an error? or an early return?

Reply:
I raised the level to error